### PR TITLE
TS Assignment

### PR DESCRIPTION
**Pull request by emtec-devyani-chaudhari**
Completed Questions: Q1, Q2, Q3, Q6, Q7, Q8, Q9, Q11, Q12, Q13, Q14, Q15.
Remaining Questions : Q4, Q5, Q10

### CODE REVIEW

File: TS-Assignment/Que4.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// write typescript code with following requirements (design proper classes). The requirements have
+// suggestions for fields and methods, you are free to add more as per your thinking
+// Bank Account
```
Comment (rishikesh-dhanawade-emtec) [inline TS-Assignment/Que4.ts:2]: where is Bank Account class?

**Review by rishikesh-dhanawade-emtec — CHANGES_REQUESTED** [review]
check comment

File: TS-Assignment/Que4.ts (hunk 1 above)
Comment (emtec-devyani-chaudhari) [reply TS-Assignment/Que4.ts:2]: Done

**Review by rishikesh-dhanawade-emtec — APPROVED** [review]
approved

**Pull request merged**